include/wx/uilocale.h

```diff
@@ -28,6 +28,13 @@ enum
     wxCompare_CaseInsensitive = 1
 };
 
+// Flags for wxLocaleIdent::RemoveLikelySubtags().
+enum
+{
+    wxSubtags_FavorRegion = 0,
+    wxSubtags_FavorScript = 1
```

Comment:
We tend to use British spelling, so I'd call them

```suggestion
    wxSubtags_FavourRegion = 0,
    wxSubtags_FavourScript = 1
```

but it's not that important, especially if they can remain private.